libsolidity/parsing/Parser.h

```diff
@@ -153,8 +153,15 @@ class Parser: public langutil::ParserBase
 	);
 	ASTPointer<Expression> parsePrimaryExpression();
 	std::vector<ASTPointer<Expression>> parseFunctionCallListArguments();
-	std::pair<std::vector<ASTPointer<Expression>>, std::vector<ASTPointer<ASTString>>> parseFunctionCallArguments();
-	std::pair<std::vector<ASTPointer<Expression>>, std::vector<ASTPointer<ASTString>>> parseNamedArguments();
+
+	using FunctionCallArguments = std::tuple<
```

Comment:
Does it make sense to define this as a structure to make further processing easier and less error prone?